templates/template-app/main/CMakeLists.txt

```diff
@@ -2,7 +2,8 @@
 set(COMPONENT_REQUIRES )
 set(COMPONENT_PRIV_REQUIRES )
```

Comment:
Same for these lines? (sorry, i didn't write a comment for each line in the previous review!)